apps/builder/app/services/token.server.ts

```diff
@@ -0,0 +1,178 @@
+import jwt from "@tsndr/cloudflare-worker-jwt";
```

Comment:
Not `jose`?

Reply:
Yep, jose is a little bit harder. This is small and simple, anyway in case of using assymetric keys signing we will switch on jose.